frontend/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,41 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { AddCompanyComponent } from './components/add-company/add-company.component';
+import { AddExchangeComponent } from './components/add-exchange/add-exchange.component';
+import { AddIpoComponent } from './components/add-ipo/add-ipo.component';
+import { CompanyComponent } from './components/company/company.component';
+import { ComparisonComponent } from './components/comparison/comparison.component';
+import { ExcelDataComponent } from './components/excel-data/excel-data.component';
+import { ExchangeComponent } from './components/exchange/exchange.component';
+import { HomeComponent } from './components/home/home.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405405)
```suggestion
import { HomeComponent } from "./components/home/home.component";
```